rtrlib/rtr_mgr.h

```diff
@@ -252,12 +254,12 @@ enum pfx_rtvals rtr_mgr_validate(struct rtr_mgr_config *config, const uint32_t a
  * @param direction The direction to check; upstream or downstream
  * @param[out] result The result of the AS path validation, i.e., whether the AS path is
  *                    considered valid, invalid, or whether the validation status is unknown
- * @return ASPA_SUCCESS on success
+ * @return RTR_ASPA_SUCCESS on success
  * @return Any other `aspa_status` code depending on the error.
  */
-enum aspa_status rtr_mgr_verify_as_path(struct rtr_mgr_config *config, uint32_t as_path[],
-							     size_t len, enum aspa_direction direction,
-							     enum aspa_verification_result *result);
+enum rtr_aspa_status rtr_mgr_verify_as_path(struct rtr_mgr_config *config, uint32_t as_path[], size_t len,
```

Comment:
maybe for consistency we also should name this `rtr_mgr_aspa_validate`, this is less descriptive but more consistent with the other names.